data/data/bootstrap/baremetal/files/etc/containers/systemd/ironic-dnsmasq.container

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=DHCP Service for Provisioning Network
+BindsTo=ironic-volume.service
+Requires=build-ironic-env.service provisioning-interface.service
```

Comment:
You probably want After= in addition to Requires=

Reply:
Fixed.